ddclient.in

```diff
@@ -1508,7 +1508,7 @@ sub _read_config {
     # Configuration line format after comment and continuation
     # removal:
     #
-    #   [opt=value, ...] [host[, ...] [login [password]]]
+    #   [opt=value, ...] [host[, ...]]
```

Comment:
We should keep this, since it is technically still supported.

Reply:
Removed that commit